src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
-body {
-  margin: 0;
+@import "./utils/_variables";
+@import "./utils/normalize";
+
+@function getShift($cell-size, $gap-size, $value) {
+  @return ($value * $cell-size) + ($value * $gap-size);
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
You have already introduced variable for `gap-size` use it here

Reply:
done.